### DISCUSSION TITLE
Documentation Link is nor working

### DISCUSSION THREAD

**Discussion by prashantraval91185** [jasonhaxstuff/popyt#474] [General]
Hi,

Can you please provide a documentation link? if it is updated to a new location.

The given URL is not working at all https://bbothell.rocks/popyt/docs/classes/_index_.youtube.html

**Comment by brandonbothell** [comment]
The [new link](https://jasonhaxstuff.github.io/popyt/docs/classes/_index_.youtube.html) has been added. Thanks for the heads up.

**Reply by prashantraval91185** [reply]
Thanks a lot, Yes, the new link is perfect, now I can able to do more work with API. Thanks again for the very fast response.

**Comment by brandonbothell** [comment]
I updated the documentation, **[check it out here](https://brandonbothell.github.io/popyt/)!**

https://github.com/brandonbothell/popyt/discussions/506
